### PR TITLE
Add TravisCi integrations

### PR DESCRIPTION
**Pull request by DrEdi**
* [x] Configure Travis CI to run pre-commit utility under python 3.5: 
(.travis.yml)
```yaml
language: python
python: 3.5
cache: pip
before_install: pip install -U pre-commit
script: pre-commit run --all-files
```
* [x] Add badge Build Status stating `master` in its URL, not current branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# django-girls-blog-DrEdi
\ No newline at end of file
+# django-girls-blog-DrEdi
+https://travis-ci.org/kpi-web-guild/django-girls-blog-DrEdi.svg?branch=master
```
Comment (webknjaz) [inline README.md]: There's a markdown version of the button showing image and making it a link. Use it

**Review by webknjaz — CHANGES_REQUESTED** [review]
You have to fix the files, so that Travis CI build won't be failing. I guess you also forgot to install pre-commit for your local repo with command `pre-commit install`. Try doing this and the checks will also run locally.

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# django-girls-blog-DrEdi
\ No newline at end of file
+[![Build Status](https://travis-ci.org/kpi-web-guild/django-girls-blog-DrEdi.svg?branch=feature%2Ftravis-ci-integration)](https://travis-ci.org/kpi-web-guild/django-girls-blog-DrEdi)
```
Comment (webknjaz) [inline README.md]: It is common to target `master` branch in a CI badge, please change this :)

**Review by webknjaz — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (DrEdi) [reply README.md]: Do not really understand what's wrong.
I had to do this from master, right?
But when I just wanted to fix it(add it in master, after come to `feature/travis-ci-integration` and push it git said that 'Everything is up-to-date'

Comment (webknjaz) [reply README.md]: `?branch=master`

File: README.md [hunk 3]
```diff
@@ -1 +1 @@
-# django-girls-blog-DrEdi
```
Comment (webknjaz) [inline README.md:1]: Add badge for `master` branch set in the URL

**Review by webknjaz — CHANGES_REQUESTED** [review]

**Comment by webknjaz** [comment]
Please re-check [common commit practices](https://github.com/kpi-web-guild/django-girls-blog-DrEdi/pull/3#issuecomment-265715633). Also, please read info about doing squash via `git rebase --interactive`, so that you could rewrite your feature branches and make their history easy-to-read and well-understandable.

**Review by webknjaz — APPROVED** [review]

**Pull request merged**